src/calendar/day/custom/index.js

```diff
@@ -41,6 +41,7 @@ class Day extends Component {
   }
 
   render() {
+    const {theme} = this.props;
```

Comment:
This is the syntax for multiple assignment, but you're only assigning "theme" - just do "const theme = this.props;"